packages/web/src/scss/components/Collapse/README.md

```diff
@@ -0,0 +1,80 @@
+# Collapse
+
+Example usage with button:
+
+```html
+<button type="button" role="button" data-toggle="collapse" data-target="CollapseExample">trigger</button>
+<div id="CollapseExample" class="Collapse">
+  <div class="Collapse__content">content</div>
+</div>
+```
+
+Usage with link:
+
+```html
+<a role="button" href="javascript:void(0)" data-toggle="collapse" data-target="CollapseExample"> trigger </a> ...
```

Comment:
I prefer setting the href to the ID of the element (and preventscroll in JS). What is your opinion?

Reply:
Good point, but in the description it was just that it should be on `data-target` and this is the example for anchor element. You cannot use `href` for the `button` element. So i think this will cover both elements.